Design/UseCases.md

```diff
@@ -0,0 +1,80 @@
+  * Actors: User
+  * BR3
+* **UC7** Create pitch pages.
+  * This is a user case because creating pitch pages is an important part of the software. Founders create pitch pages to find potential investors or developers for their project. The pitch page hosts the information that will attract these users to the project.
+  * To create a pitch page a founder will navigate to the button on the home page that says "Create Pitch". This will redirect them to the pitch creation page. Here, they'll have to enter the name of their pitch, details, and keywords for others to be able to find it. 
+  * Actors: Founder
+  * BR3
+
+* **UC8** Interchange between user roles.
+  * Interchanging between user roles is an important use case because users may have to switch between different roles. For example, a founder for a project may be a developer on another project and/or on the same project. Users need to be able to switch between roles so they can take on the different roles that are required of them per project.
+  * The user will be able to select the different roles they will have on a project-by-project basis, or the roles will be given to them by the project founder if they are not the founder.
+  * Actors: User
+  * BR4
+
+* **UC9** Hold investor payments.
+  * This use case is critical to the software. Holding investor payments is one of the most important tasks the software has since it serves as an intermediary between the different users. It keeps the investor payments safe so everyone is satisfied and the money goes to who its supposed to go to during or after the projects lifespan.
+  * The investors give the money to the system and holds it until the system is told by the founder to disperse the money to the necessary parties.
+  * Actors: System
+  * BR5
+* **UC10** Finish a given project.
+  * This use case is necessary because the project should eventually be completed. Once the project is finished funds can be dispersed. The end goal of the project is to eventually reach this point.
+  * In the project page the developer will find the "Finish" button within the project settings. Clicking it will finish the project.
+  * Actors: Developer
+  * BR5
+
+* **UC11** Negotiate terms of a pitch.
+  * The ability to negotiate terms is important because it lets the different users involved in the process come to an agreement on project details. Once the different users come to agreement the pitch is complete and the project can move forward. Without this functionality, the users would find it difficult to come to an appropriate agreement on individual terms for the pitch. One way they will be able to negotiate is through the direct message functionality.
+  * To negotiate terms of a pitch the users will navigate to the direct messages page. Here, they can choose who to negotiate with by sending a message to them.
+  * Actors: User
+  * BR6
+* **UC12** Observe associated parties with the pitch.
```

Comment:
For this one I would go in depth about how it doesn't matter which user you are when observing associated parties through using the search functionality on the home page